crates/cli-support/src/js/mod.rs

```diff
@@ -377,13 +377,14 @@ impl<'a> Context<'a> {
             // In `--target no-modules` mode we need to both expose a name on
             // the global object as well as generate our own custom start
             // function.
+            // `document.currentScript` property can be null in browser extensions
             OutputMode::NoModules { global } => {
                 js.push_str("const __exports = {};\n");
                 js.push_str("let script_src;\n");
                 js.push_str(
                     "\
                     if (typeof document !== 'undefined') {
-                        script_src = new URL(document.currentScript.src, location.href).toString();
+                        script_src = new URL((document.currentScript && document.currentScript.src) ? document.currentScript.src : '', location.href).toString();
```

Comment:
We should probably just leave `script_src` as `undefined` in this case, so the check could just be moved into the `if` condition.

Reply:
1. tested leaving it `undefined` manually
2. changed the if-statement as requested